sdks/go/pkg/beam/core/runtime/xlangx/expansionx/download.go

```diff
@@ -382,7 +382,14 @@ func jarExists(jarPath string) bool {
 	return err == nil
 }
 
-func getPythonVersion() (string, error) {
+// GetPythonVersion returns the Python version to use. It checks for
+// env variable PYTHON_PATH and returns that it if set.
+// If no PYTHON_PATH is defined then it checks for python or python3
```

Comment:
Let's modify the error message on line 399 to say sth like: 

no python installation found. If you use a custom container image, please verify that python or python3 command launches the interpreter  or specify the full path to the interpreter file in PYTHON_PATH environment variable.